Directory.Build.props

```diff
@@ -19,18 +19,21 @@
     <HtmlAgilityPackVersion>1.11.57</HtmlAgilityPackVersion>
     <HumanizerCoreVersion>2.14.1</HumanizerCoreVersion>
     <HumanizerVersion>2.14.1</HumanizerVersion>
+    <MicrosoftBclAsyncInterfacesVersion>8.0.0</MicrosoftBclAsyncInterfacesVersion>
     <MicrosoftCodeAnalysisCommonVersion>4.12.0</MicrosoftCodeAnalysisCommonVersion>
     <MicrosoftDataSqlClientVersion>5.2.2</MicrosoftDataSqlClientVersion>
     <MicrosoftEntityFrameworkCoreDesignVersion>9.0.0</MicrosoftEntityFrameworkCoreDesignVersion>
     <MicrosoftEntityFrameworkCoreSqlServerVersion>9.0.0</MicrosoftEntityFrameworkCoreSqlServerVersion>
     <MicrosoftIdentityClientVersion>4.65.0</MicrosoftIdentityClientVersion>
     <MicrosoftNETTestSdkVersion>17.12.0</MicrosoftNETTestSdkVersion>
     <NewtonsoftJsonVersion>13.0.3</NewtonsoftJsonVersion>
+    <SystemCollectionsImmutableVersion>8.0.0</SystemCollectionsImmutableVersion>
     <SystemDrawingCommonVersion>9.0.0</SystemDrawingCommonVersion>
     <SystemReactiveVersion>6.0.0</SystemReactiveVersion>
+    <SystemReflectionMetadataVersion>8.0.0</SystemReflectionMetadataVersion>
     <SystemSecurityCryptographyPkcsVersion>9.0.0</SystemSecurityCryptographyPkcsVersion>
     <SystemSecurityCryptographyXmlVersion>9.0.0</SystemSecurityCryptographyXmlVersion>
-    <SystemTextJsonVersion>9.0.0</SystemTextJsonVersion>
+    <SystemTextJsonVersion>8.0.0</SystemTextJsonVersion>
```

Comment:
I think this may trigger component governance alerts. I am guessing it may be best to go back to the minor version that we were on before we bumped everything to 9.*